src/ShowComand.php

```diff
@@ -28,11 +28,16 @@ public function execute(InputInterface $input, OutputInterface $output)
             ->addOption('type', 'type')
             ->url;
 
-        $movieInfo = $this->getMovieInfo($url, $output);
+        [$movieInfo, $statusCode] = $this->getMovieInfo($url, $output);
+
+        if (! $movieInfo || $movieInfo->Response !== 'True') {
```

Comment:
lets move this to inside the getMovieInfo() method, again with the idea of encapsulating all the API related things inside one method

> $movieInfo->Response !== 'True'